docs/operator/configuration.md

```diff
@@ -17,6 +17,7 @@ Configuration of the operator's Pod is done via environment variables at startup
 | `OPERATOR_VULNERABILITY_SCANNER_ENABLED`                     | `true`               | The flag to enable vulnerability scanner                                                                                                                                                                     |
 | `OPERATOR_CONFIG_AUDIT_SCANNER_ENABLED`                      | `true`               | The flag to enable configuration audit scanner                                                                                                                                                               |
 | `OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS` | `false`              | The flag to enable vulnerability scanner to only scan the current revision of a deployment                                                                                                                   |
+| `OPERATOR_ENABLE_SCAN_JOB_IN_SAME_NAMESPACE`                 | `false`              | The flag to enable running scan jobs in the same namespace of scanned resource. Currently this option is enabled for only vulnerability scanning                                                             |
```

Comment:
Use common prefix for vulnerability scanner configs `OPERATOR_VULNERABILITY_SCANNER_`